examples/brownian_motion.dx

```diff
@@ -1,5 +1,5 @@
 
-include "plot.dx"
+import plot
```

Comment:
Have you thought about supporting qualified imports, like `import plot.matshow`? Definitely seems like a non-essential feature, it'd be relevant for code size optimization and faster compilation (avoid deserializing unused parts of the imported module).

Reply:
Yes, definitely, and similarly on the export side, where you restrict what's exported to achieve safety guarantees. But it's low priority for now.

Reply:
Also, it's a minor note, but I _absolutely hate_ when languages forbid me from accessing internal members. Of course it's good to have a way to distinguish those syntactically, because it's less safe, but sometimes you just want to get something done, without waiting for others to re-export certain functionality!